plugins/azure/openai/accountManagedIdentity.js

```diff
@@ -0,0 +1,55 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'OpenAI Account Managed Identity Enabled',
+    category: 'AI & ML',
+    domain: 'Machine Learning',
+    description: 'Ensures a system or user assigned managed identity is enabled to authenticate to Azure OpenAI accounts.',
+    more_info: 'Enabling managed identity for Azure OpenAI accounts automates credential management, enhancing security by avoiding hard-coded credentials and simplifying access control to Azure services.',
+    recommended_action: 'Enable system or user-assigned identities for all Azure OpenAI accounts.',
+    link: 'https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/managed-identity',
+    apis: ['openAI:listAccounts'],
+    realtime_triggers: ['microsoftcognitiveservices:accounts:write','microsoftcognitiveservices:accounts:delete'],
+
+    run: function(cache, settings, callback) {
+        const results = [];
+        const source = {};
+        const locations = helpers.locations(settings.govcloud);
+
+        async.each(locations.openAI, function(location, rcb) {
+            const accounts = helpers.addSource(cache, source,
+                ['openAI', 'listAccounts', location]);
+
+            if (!accounts) return rcb();
+
+
+            if (accounts.err || !accounts.data) {
+                helpers.addResult(results, 3, 'Unable to query OpenAI accounts: ' + helpers.addError(accounts), location);
+                return rcb();
+            }
+
+            if (!accounts.data.length) {
+                helpers.addResult(results, 0, 'No existing OpenAI accounts found', location);
+                return rcb();
+            }
+
+            for (let account of accounts.data) {
+
+                if (account.identity && account.identity.type && 
```

Comment:
it looks good but i think its fine without adding the checks of identity types like we have in other plugins

Reply:
if we enable then disable it then it returns type: "none"
<img width="667" alt="Screenshot 2024-02-19 at 3 35 47 PM" src="https://github.com/aquasecurity/cloudsploit/assets/87388442/51f66026-fe25-403b-9584-05c706ebc8f3">